packages/webapp/pages/onboarding.tsx

```diff
@@ -355,11 +357,11 @@ export function OnboardPage(): ReactElement {
 
     const params = new URLSearchParams(window.location.search);
     const afterAuth = params.get(AFTER_AUTH_PARAM);
+    const query = onboardingPapercut ? undefined : { ua: 'true' };
```

Comment:
What is this again?

Reply:
Ah, whether we should display the UserAcquisition card from the Feed.